### DISCUSSION TITLE
Liveness and Readiness Probes support in EmqxBroker

### DISCUSSION THREAD

**Discussion by burhanshaikh** [emqx/emqx-operator#171] [Q&A] [answered]
At the moment I am not able to specify Liveness and Readiness Probes in EmqxBroker Custom Resource.
I want to specify custom initialDelaySeconds and other fields under the probe.
It is not defined in the CRD, or am I missing something? If so can someone please share a sample CR with those fields?

Using chart version: emqx-operator-1.0.3 and app version: 1.1.5

**Comment by Rory-Z** [comment, marked as answer]
Hi, @burhanshaikh 
Unfortunately EmqxBroker Custom Resource does not support custom Liveness and Readiness Probes at the moment, perhaps we can add this feature in the next release

**Comment by Rory-Z** [comment]
See https://github.com/emqx/emqx-operator/pull/180

**Comment by Rory-Z** [comment]
Hi, @burhanshaikh EMQX Operator 1.2 will coming soon, please check https://github.com/emqx/emqx-operator/pull/231
In v1beta3, we support set up `readinessProbe/livenessProbe/startupProbe` by `.spec.emqxTemplate` in EMQX Custom Resource

**Reply by burhanshaikh** [reply]
Hey @Rory-Z Thank you for this!
